documentation/modules/security/proc-securing-kafka.adoc

```diff
@@ -61,7 +61,7 @@ If you are configuring an external listener, the configuration is dependent on t
 [source,shell,subs=+quotes]
 kubectl apply -f _<kafka_configuration_file>_
 +
-The Kafka cluster is configured with a Kafka broker listener using TLS authentication.
+The Kafka cluster is configured with a Kafka broker listener using mTLS authentication.
 +
 A service is created for each Kafka broker pod.
```

Comment:
Not related to this PR. But this is actually not true.

Reply:
We say this in two procedures. Just remove?

Reply:
What is the other procedure? It is incorrect here. But might be valid elsewhere.

Reply:
[Line 122](https://github.com/strimzi/strimzi-kafka-operator/pull/7479/files#diff-861669170e7c4c935cef0c52be158c1496ef914f1b8d9e9dd9b8f40abf377ed9R122) (and related changes) in [proc-deploy-setup-external-clients.adoc](https://github.com/strimzi/strimzi-kafka-operator/pull/7479/files#diff-861669170e7c4c935cef0c52be158c1496ef914f1b8d9e9dd9b8f40abf377ed9)

Same config, but with unspecified listener type.